rakefile.netstd.rb

```diff
@@ -0,0 +1,405 @@
+end
+
+desc "create the analyzer nuget package"
+exec :pack => [:build, output] do |cmd|
+  cmd.command = nuget_command
+  cmd.parameters "pack #{analyzer_nuspec} -Version #{version}#{version_suffix} -OutputDirectory #{output}"
+end
+
+def create_release(client, repo, milestone, release_version, release_body, release_issue_body, release_issue_labels)
+  release_description = release_version + ' release'
+  puts "Creating issue '#{release_description}'..."
+  issue = client.create_issue(
+    repo,
+    release_description,
+    release_issue_body,
+    :labels => release_issue_labels,
+    :milestone => milestone.number
+    )
+  puts "Created issue \##{issue.number} '#{release_description}'."
+
+  puts "Creating release '#{release_version}'..."
+  client.create_release(
+    repo,
+    release_version,
+    :name => release_version,
+    :draft => true,
+    :body => release_body
+    )
+  puts "Created release '#{release_version}'."
+end
+
+def print_vars(variables)
+
+  scalars = []
+  vectors = []
+
+  variables.each { |name, value|
+    if value.respond_to?('each')
+      vectors << [name, value.map { |v| v.to_s }]
+    else
+      string_value = value.to_s
+      lines = string_value.lines
+      if lines.length > 1
+        vectors << [name, lines]
+      else
+        scalars << [name, string_value]
+      end
+    end
+  }
+
+  scalar_name_column_width = scalars.map { |s| s[0].length }.max
+  scalars.each { |name, value|
+    puts "#{name}:#{' ' * (scalar_name_column_width - name.length)} #{value}"
+  }
+
+  puts
+  vectors.select { |name, value| ![
+                                   'release_body',
+                                   'release_issue_common_steps',
+                                   'next_version_steps',
+                                   'release_issue_labels'
+                                  ].include? name }.each { |name, value|
+    puts "#{name}:"
+    puts value.map {|v| "  " + v }
+    puts ""
+  }
+end
+
+def run_msbuild(solution, target, command, packages_dir = nil)
+  packages_dir_option = "/p:NuGetPackagesDirectory=#{packages_dir}" if packages_dir
+  cmd = Exec.new
+  cmd.command = command
+  cmd.parameters "#{solution} /target:#{target} /p:configuration=Release /nr:false /verbosity:minimal /nologo /fl /flp:LogFile=artifacts/logs/#{target}.log;Verbosity=Detailed;PerformanceSummary #{packages_dir_option}"
+  cmd.execute
+end
+
+def run_tests(test_assemblies, command, result_dir)
+  test_assemblies.each do |test_assembly|
+    result_file = File.expand_path(File.join(result_dir, File.basename(test_assembly, '.dll') + '.TestResults.xml'))
+
+    xunit = XUnitTestRunner.new
+    xunit.command = command
+    xunit.assembly = test_assembly
+    xunit.options '-noshadow', '-nologo', '-notrait', '"explicit=yes"', '-xml', result_file
+    xunit.execute
+  end
+end
+
+def run_netstd_tests(test_directories, result_dir)

```

Comment:
A nitpick, but this isn't really a method for running netstd tests. The test target happens to be netstd, but the test project is netcoreapp, which is why it uses the `dotnet` CLI. Perhaps this would be better named as `run_dotnet_tests`? Or `run_tests_dotnet`?